client/src/components/form.js

```diff
@@ -12,47 +13,77 @@ const useStyles = makeStyles((theme) => ({
       color: 'white'
     },
   },
+
+  uiaas: {
+    backgroundColor: "inherit",
+  },
+
+  card: {
+    backgroundColor: "inherit",
+  },
+
+  cardheader: {
+    color: "whitesmoke",
+    fontSize: "20px",
+  },
+
+  textfield: {
+    color: "whitesmoke",
+    fontSize: "20px",
+  },
 }));
 
 
 export default function Form() {
-    const [formNum, setFormNum] = useState(0)
-    const [customers, setCustomers] = useState([0]) 
-    const classes = useStyles();
-
-    useEffect(() => {
-        // Axios Method
-        Axios({
-            method: 'GET',
-            url: '/customers',
-        }).then(res => 
-            res.data,
-        ).then(customers =>     
-            setCustomers(customers)
-        );
-    }, [formNum]);
-
-    function displayCustomers() {
-        return customers.map((customer) => 
-        <div>
-            <TextField id="filled-basic" label={customer.username} variant="filled"/>
-        </div>   
-        );
-        }
-
-        return (
-            <div>
-                <form className={classes.root}>
-                <h2>Customers</h2>
-                    <div>
-                        {displayCustomers()}
-                    </div>
-                </form>
-                <Button 
-                    onClick={() => { setFormNum( formNum + 1) }}
-                    variant="contained" color="primary" href="">
-                    Next Page
-                </Button>
-            </div>
-        );
-    }
\ No newline at end of file
+  const [formNum, setFormNum] = useState(0)
+  const [customers, setCustomers] = useState([0]) 
+  const [forms, setForms] = useState([0]) 
+  const classes = useStyles();
+
+  useEffect(() => {
+      // Axios Method
+      Axios({
+          method: 'GET',
+          url: '/applications/UCDavis/forms/b',
+      }).then(res => 
+          res.data,
+      ).then(forms =>     
+          setForms(forms)
+      );
+  }, [formNum]);
+
+  function displayForms() {
+    return (
+      Object.getOwnPropertyNames(forms).map((fieldName) => 
+        <TextField color="whitesmoke" className="textfield" label={fieldName} variant="filled"/>
```

Comment:
Since the default value for a TextField's color is set to "whitesmoke" earlier in this file, do you need to re-set it here?